pages/isoman.tsx

```diff
@@ -0,0 +1,53 @@
+import { BackButton } from "~/components/layout/back-button";
+import { Page } from "~/components/layout/page";
+import { PageContent } from "~/components/layout/page-content";
+import { PageHeader } from "~/components/layout/page-header";
+import StackedLink from "~/components/layout/stacked-link";
+import config from "~/lib/config";
+import isoman from "~/lib/isoman-contents";
+
+import { NextSeo } from "next-seo";
+
+const meta = {
+  title: `Pedoman Isolasi Mandiri | ${config.site_name}`,
+};
+
+export default function Isoman() {
+  const { isoman_contents } = isoman;
+
+  return (
```

Comment:
Please write some integration tests for this page.
You can look at the existing tests for the `FAQ` page as a reference.

Reply:
I have a problem with generating the mock data. Then only use the existing data.
It only checks for the components render both on isoman page and the stacked link component.
![image](https://user-images.githubusercontent.com/3627108/127149989-d3623184-ee1a-4cac-ac57-6706618e4d7c.png)

Reply:
Please make sure that you pushed your changes, @syauqy, so I can help with the tests, as I did with some other PRs having trouble with the tests.
If you don't push your changes, and I added more tests, there would be conflicts.

Reply:
Eh? sorry sorry, I'll push it now. Also with the slug changes.